core/Makefile

```diff
@@ -314,11 +356,22 @@ flash_combine: $(PRODTEST_BUILD_DIR)/combined.bin ## flash combined using OpenOC
 flash_erase: ## erase all sectors in flash bank 0
 	$(OPENOCD) -c "init; reset halt; flash info 0; flash erase_sector 0 0 last; flash erase_check 0; exit"
 
+flash_erase_bootloader: ## erase bootlaoder
+	$(OPENOCD) -c "init; reset halt; flash info 0; flash erase_sector 0 $(BOOTLOADER_SECTOR_START) $(BOOTLOADER_SECTOR_END); exit"
+
+flash_erase_firmware: ## erase bootlaoder
```

Comment:
Small typo in comment - `bootlaoder` instead of `bootloader`

Reply:
fixed https://github.com/trezor/trezor-firmware/pull/3480/commits/a21292edcb44ba5daa116be4c109830febf1a144